tests/types/types.test.ts

```diff
@@ -1,7 +1,8 @@
 import css from "@eslint/css";
-import { ESLint } from "eslint";
+import type { CSSLanguageOptions } from "@eslint/css";
+// import { ESLint } from "eslint";
 
-css satisfies ESLint.Plugin;
```

Comment:
If we update `"eslint"` in package.json to `^9.23.0`, this test will pass again.